docs/quick-start-guide.md

```diff
@@ -10,6 +10,9 @@ Once you're familiar with the general idea, please start hacking...
 
 ## Quick start
 
+### Using Docker
+check [docker-guide](docs/docker-guide.md) and skip to the *Running mgmt* section
```

Comment:
Please use a full sentence with capitalization and period and move this section to the end after `Vagrant` server.